molgenis-data-import/src/main/java/org/molgenis/data/importer/emx/EmxMetaDataParser.java

```diff
@@ -492,7 +519,7 @@ private void parsePackagesSheet(Repository<Entity> repo, IntermediateParseResult
 			Tag tag = intermediateResults.getTag(tagIdentifier);
 			if (tag == null)
 			{
-				throw new IllegalArgumentException("Unknown tag '" + tagIdentifier + '\'');
+				throw new UnknownTagException(tagIdentifier);
```

Comment:
UnknownEntityException instead?

Reply:
assign

Reply:
@fdlk @tommydeboer Already fixed? One of you happened to fix this already and forgot to assign yourself to the review comment?